general-configuration-guide/boilerplate-content-css.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+title: Boilerplate content CSS
+title_nav: Boilerplate content CSS
+description_short: Learn how to set up CSS for your site to integrate TinyMCE.
+description: Learn how to set up CSS for your site to integrate TinyMCE.
+keywords: css content_css
+---
+
+Some features in {{site.productname}} that require additional CSS to be added to the page that displays the content produced by the editor. In general, to get as clean an output as possible from a rich text editor as well as to use some advanced features like captioned images, use CSS rather than inline styles to render correctly.
```

Comment:
`s/that require/require/`?

Reply:
Second sentence is also very long. It feels like it needs a comma or should be broken up.

Reply:
that whole paragraph was horrible (I just brought it over as-is). Significant revisions made.

Reply:
May be fixed by https://github.com/tinymce/tinymce-docs/pull/1605/commits/64c3e93fc457c41ffb4b02d1213bcbb580b98674